src/integration.test.ts

```diff
@@ -58,51 +81,40 @@ describe("Integration Test", (): void => {
     });
   });
 
-  const mockedExec = async (load: boolean, command: string): Promise<void> => {
-    const stdout = "standard output from Bash command";
-    const stderr = "error output from Bash command";
-    child_process.exec.mockImplementationOnce(
-      (_command: any, _options: any, callback: any): any => {
-        callback(null, { stdout, stderr });
-      }
-    );
-
-    await (load ? docker.loadDockerImages() : docker.saveDockerImages());
-
+  const execAsserts = (command: string, n: number): void => {
     expect(core.getInput).nthCalledWith(1, "key", { required: true });
-    expect(core.info).nthCalledWith<[string]>(1, command);
     expect(child_process.exec).lastCalledWith(
       command,
       EXEC_OPTIONS,
       expect.anything()
     );
-    expect(core.info).lastCalledWith(stdout);
-    expect(core.error).lastCalledWith(stderr);
+    expect(core.info).nthCalledWith<[string]>(n, command);
+    expect(core.error).lastCalledWith(STDERR);
     expect(core.setFailed).not.toHaveBeenCalled();
   };
 
   test("cache misses, then hits", async (): Promise<void> => {
     // Attempt first cache restore.
+    const listCommand =
+      'docker image list --format "{{ .Repository }}:{{ .Tag }}"';
```

Comment:
I also fixed this one.  I think a linter is changing it back.  Let me know if there is a setting or something I can change so that this doesn't happen.